files/scripts/syslog_counter.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
```

Comment:
>python3 [](http://example.com/codeflow?start=15&length=7)

For high performance, and low memory consumption, suggest use Rust for new daemon/long running process.

Reply:
Fixed, convert to rust